src/scopes/grants/grantStatus.js

```diff
@@ -0,0 +1,38 @@
+import { Op } from 'sequelize';
+import { sequelize } from '../../models';
+
+const grantStatusSql = (grantStatus, notWithin) => {
+  let where = null;
+  if (grantStatus === 'active') {
+    where = `"Grants"."status" = '${notWithin ? 'Inactive' : 'Active'}' AND "Grants"."cdi" = false`;
+  } else if (grantStatus === 'inactive') {
+    where = `"Grants"."status" = '${notWithin ? 'Active' : 'Inactive'}' AND "Grants"."cdi" = false`;
+  } else {
+    where = `"Grants"."cdi" = ${notWithin ? 'false' : 'true  AND "Grants"."status" = \'Active\''}`;
```

Comment:
Same as above

Reply:
Ok I spoke with Kelly for now only show we will keep as is with active, inactive, and active cdi.